Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1374,6 +1374,17 @@ private void LogCommandLineTooLong(SandboxedProcessInfo info)
 
                         if (m_pip.RetryExitCodes.Contains(result.ExitCode) && m_remainingUserRetryCount > 0)
                         {
+                            Tuple<AbsolutePath, Encoding> encodedStandardError = null;
+                            Tuple<AbsolutePath, Encoding> encodedStandardOutput = null;
+                            if (await TrySaveAndLogStandardOutputAsync(result))

```

Comment:
>TrySaveAndLogStandardOutputAsync [](start = 38, length = 32)

`TrySaveAndLogStandard*` logs the content of the file. So, why do we need to log it again in `PipExecutor.cs` ?